react/src/lib/components/WellLogViewer/MapAndWellLogViewer.stories.jsx

```diff
@@ -4,6 +4,8 @@ import { MapAndWellLogViewer } from "./MapAndWellLogViewer";
 
 import exampleData from "../../../demo/example-data/deckgl-map.json";
 import { colorTables } from "@emerson-eps/color-tables";
+//import { ColorTable } from "./components/ColorTableTypes";
```

Comment:
Maybe we could remove this and other disabled code.

Reply:
removed